# Whole-Genome Sequencing Accurately Identifies resistance to Extended-Spectrum β-Lactams for Major Gram-Negative Bacterial Pathogens

## Authors: Sarah Shebl (@Sarah50), Akeemat Ayinla (@That\_Hakeemah), Frederick Allou (@Freddie), Duong Gia Khanh (@Khanh)

### Video link: <https://www.linkedin.com/posts/sarah-shebl-a8159b203_wgs-antimicrobialresistance-hackbio-activity-7238132635984683008-8-37?utm_source=share&utm_medium=member_desktop>


**Introduction**
\
The study explores the effectiveness of whole-genome sequencing (WGS) in predicting antimicrobial resistance (AMR) compared to traditional methods such as PCR as a genotypic method and broth microdilution (BMD) as a phenotypic method. This research addresses the growing challenge of β-Lactam resistance among gram-negative bacteria, which poses a serious threat to the treatment of neutropenic cancer patients. By evaluating WGS alongside PCR and BMD, the study highlights how WGS can provide a more comprehensive understanding of resistance mechanisms, potentially leading to better-informed treatment decisions in this high-risk population.\
\
**Methodology**\
The study used Illumina MiSeq WGS and BMD to analyze 90 bloodstream isolates from common gram-negative bacteria (_Escherichia coli, Klebsiella pneumoniae, Pseudomonas aeruginosa _and_ Enterobacter cloacae_) in cancer patients. The study aimed to estimate resistance to four β-Lactams (ceftazidime, cefepime, piperacillin-tazobactam (P/T), and meropenem) commonly used in the treatment of neutropenic cancer patients with fever.\
\
**Results and discussion**

**Advantages of WGS over PCR**

WGS offers significant advantages over PCR, as it allows us to analyze the entire genetic makeup of bacteria, enabling us to identify a broader range of specific resistance mechanisms with remarkable accuracy. While PCR is effective for identifying specific β-lactamase resistance genes, it  does not account for other resistance mechanisms such as porin mutations, efflux pumps and the activation of chromosomal β-lactamases genes. The study identified two main categories of resistance mechanisms: the acquisition of exogenous β-lactamases, such as CTX-M and NDM enzymes, and chromosomal gene inactivation events, like mutations in the OprD porin gene in _Pseudomonas aeruginosa_. It was found that only 65% of the predicted AMR events were linked to exogenous β-lactamase genes that could be detected using standard PCR techniques. This highlights a critical limitation of PCR, as it may not capture all resistance mechanisms, particularly those involving chromosomal mutations or other genetic alterations.\
\
**Comparison to BMD**

BMD is considered the gold standard for determining antimicrobial susceptibility; however, it is time-consuming and resource-intensive, making it less practical for routine clinical use. In contrast, WGS provides rapid results and can be performed in a single comprehensive procedure. The study found that WGS predictions agreed with BMD results in 93% of cases, demonstrating its reliability. Moreover, WGS showed a sensitivity of 0.87 and specificity of 0.98 compared to BMD, with positive and negative predictive values of 0.97 and 0.91, respectively. This performance indicates that WGS can serve as a robust alternative to traditional methods.

**Conclusion**

Integrating WGS into clinical microbiology can significantly enhance antimicrobial resistance (AMR) detection, particularly in vulnerable cancer patients. This study highlights WGS's superiority over PCR and BMD for predicting AMR, suggesting that its advancement could transform AMR detection and improve patient outcomes in infection management

**References**

Shelburne, S. A., Kim, J., Munita, J. M., Sahasrabhojane, P., Shields, R. K., Press, E. G., ... & Greenberg, D. E. (2017). Whole-genome sequencing accurately identifies resistance to extended-spectrum β-lactams for major gram-negative bacterial pathogens. _Clinical Infectious Diseases_, _65_(5), 738-745.

 
